client/simplestreams_images.go

```
@@ -21,6 +21,11 @@ func (r *ProtocolSimpleStreams) GetImages() ([]api.Image, error) {
 	return r.ssClient.ListImages()
 }
 
+// GetFilteredImages returns a filtered list of available images as Image structs
+func (r *ProtocolSimpleStreams) GetFilteredImages(filters []string) ([]api.Image, error) {
```

I think this should return an error instead.